### PR TITLE
 modify the node qos ensurance definition

### PR DESCRIPTION
**Pull request by yan234280533**

### CODE REVIEW

File: ensurance/v1alpha1/types.go [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
 	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	corev1 "k8s.io/api/core/v1"
+)
+
+type AvoidanceActionStrategy string
+
+const (
+	// AvoidanceActionStrategyNone do the action when the rules triggered.
+	AvoidanceActionStrategyNone AvoidanceActionStrategy = "None"
```
Comment (mfanjie) [inline ensurance/v1alpha1/types.go:13]: should this be `auto` instead of `none`?

File: ensurance/v1alpha1/types.go [hunk 2]
```diff
@@ -117,59 +127,31 @@ type ObjectiveEnsurance struct {
 	// Metric rule define the metric identifier and target
 	MetricRule *MetricRule `json:"metricRule,omitempty"`
 
-	// How many times the rule is reach, to trigger action, default is 1
-	ReachedThreshold int32 `json:"reachedThreshold,omitempty"`
+	// How many times the rule is reach, to trigger avoidance action, default is 1
+	AvoidanceThreshold int32 `json:"avoidanceThreshold,omitempty"`
```
Comment (mfanjie) [inline ensurance/v1alpha1/types.go:131]: RestoredThreshold->RestoreThreshold

Comment (yan234280533) [reply ensurance/v1alpha1/types.go:134]: done~

File: ensurance/v1alpha1/types.go (hunk 1 above)
Comment (yan234280533) [reply ensurance/v1alpha1/types.go:13]: done~

**Comment by yan234280533** [comment]
The crane pr is LGTM, merged along with it. 
https://github.com/gocrane/crane/pull/92

**Pull request merged**